Only in src/rcl_orig/: COLCON_IGNORE
diff -ru src/rcl_orig/rcl/CMakeLists.txt src/rcl/rcl/CMakeLists.txt
--- src/rcl_orig/rcl/CMakeLists.txt	2020-02-20 19:47:11.994168265 +0000
+++ src/rcl/rcl/CMakeLists.txt	2020-02-20 19:49:29.063311466 +0000
@@ -30,6 +30,7 @@
 
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
+  add_link_options(-latomic)
 endif()
 
 set(${PROJECT_NAME}_sources
diff -ru src/rcl_orig/rcl/src/rcl/context.c src/rcl/rcl/src/rcl/context.c
--- src/rcl_orig/rcl/src/rcl/context.c	2020-02-20 19:47:12.024168075 +0000
+++ src/rcl/rcl/src/rcl/context.c	2020-02-20 20:07:01.048726300 +0000
@@ -38,7 +38,7 @@
     sizeof(context.instance_id_storage) >= sizeof(atomic_uint_least64_t),
     "expected rcl_context_t's instance id storage to be >= size of atomic_uint_least64_t");
   // initialize atomic
-  atomic_init((atomic_uint_least64_t *)(&context.instance_id_storage), 0);
+  // atomic_init((atomic_uint_least64_t *)(&context.instance_id_storage), 0);
   return context;
 }
 
